src/Components/MainArea/Bookshelf/Bookshelf.tsx

```diff
@@ -1,5 +1,19 @@
 import React from 'react';
+import { IBooks } from '../../../typings/IBooks';
+import { Bookcase } from './Bookcase/Bookcase';
 
-const Bookshelf = () => <div>Bookshelf</div>;
+interface IProps {
```

Comment:
`IProps` sounds like a very generic name. Let's try to find a name that indicates the actual meaning of this interface. Maybe `IBookshelfProps`?

Reply:
My reasoning for this was, I wasn't using this interface anywhere else, just for the component the interface was created for. That's why more generic IProps and IStates was sufficient. I did correct this.